db/schema.rb

```diff
@@ -162,6 +162,22 @@
     t.index ["uuid"], name: "index_identities_on_uuid", unique: true
   end
 
+  create_table "job_runs", force: :cascade do |t|
+    t.string "host", null: false
+    t.string "pid", null: false
+    t.datetime "finish_time"
+    t.string "job_name", null: false
+    t.string "result"
+    t.string "error"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["created_at"], name: "index_job_runs_on_created_at"
+    t.index ["error"], name: "index_job_runs_on_error"
+    t.index ["finish_time"], name: "index_job_runs_on_finish_time"
+    t.index ["host"], name: "index_job_runs_on_host"
+    t.index ["job_name"], name: "index_job_runs_on_job_name"
```

Comment:
if a recurring job will have a single job name rather than having sole indexes for finish_time and created_at (which would give you the global view) you probably want them in the context of the job_name.  ie ["job_name", "created_at"]

Reply:
which would answer the question when did job_name x run last?

Reply:
Good call, I'll update both